src/Core/Components/Button/FluentButton.razor

```diff
@@ -13,9 +13,9 @@
                    formmethod="@FormMethod"
                    formnovalidate="@FormNoValidate"
                    formtarget="@FormTarget"
-                   type="@(Type == ButtonType.Button ? null : Type.ToAttributeValue())"
```

Comment:
You cannot remove these tests: the default values are `button`, `rounded` and `medium`. So if the dev use one of them, the HTML code is reduced with no attribute and the result will be the same.

Reply:
If you do not specify a `Type`, the default value (now `null`) is applied and so it will not show up in the HTML. If you supply a `Type="ButtonType.Button"` then `type="button` should be in the HTML as well.

Same goes for `Shape` and `Size` .